Formula/zbctl.rb

```diff
@@ -0,0 +1,42 @@
+class Zbctl < Formula
+  desc "Distributed Workflow Engine for Microservices Orchestration"
+  homepage "https://docs.camunda.io/docs/apis-clients/cli-client/index/"
+  url "https://github.com/camunda-cloud/zeebe.git",
+      tag:      "1.2.9",
+      revision: "b55242898afe0dd93b87fa0b464f7c4d9a10bbad"
+  license "Apache-2.0"
+  head "https://github.com/camunda-cloud/zeebe.git", branch: "develop"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    commit = Utils.git_short_head
+    chdir "clients/go/cmd/zbctl" do
+      project = "github.com/camunda-cloud/zeebe/clients/go/cmd/zbctl/internal/commands"
+      ldflags = %W[
+        -w
+        -X #{project}.Version=#{version}
+        -X #{project}.Commit=#{commit}
+      ]
+      system "go", "build", "-tags", "netgo", *std_go_args(ldflags: ldflags)
+    end
+  end
+
+  test do
+    # Check status for a nonexistent cluster
+    status_error_message =
+      "Error: rpc error: code =" \
+      " Unavailable desc = connection error:" \
+      " desc = \"transport: Error while dialing dial tcp 127.0.0.1:26500: connect: connection refused\""
+    output = shell_output("#{bin}/zbctl status 2>&1", 1)
+    assert_match status_error_message, output
+    # Check version
+    expected_version = "zbctl #{version} (commit: b5524289)"
```

Comment:
This will have to be changed every time the formula is updated because the commit hash is specified. Take a look at `brigade-cli`'s test, where the commit hash is accessed in the `test` block via `stable.specs[:revision]`. This would result in something like:
```suggestion
    commit = stable.specs[:revision][0..8]
    expected_version = "zbctl #{version} (commit: #{commit})"
```

Reply:
Done! Thanks again :)